feature-libs/asm/components/asm-bind-cart/asm-bind-cart.component.ts

```diff
@@ -81,15 +94,24 @@ export class AsmBindCartComponent implements OnInit, OnDestroy {
     protected multiCartFacade: MultiCartFacade,
     protected asmBindCartFacade: AsmBindCartFacade,
     protected launchDialogService: LaunchDialogService,
-    protected savedCartFacade: SavedCartFacade
+    protected savedCartFacade: SavedCartFacade,
+    protected asmComponentService: AsmComponentService,
+    protected routing: RoutingService,
+    @Optional() protected featureConfig?: FeatureConfigService
   ) {}
 
   ngOnInit(): void {
+    this.subscribeForDeeplinkInactiveCart();
+
     this.subscription.add(
       this.activeCartFacade.getActiveCartId().subscribe((response) => {
         this.activeCartId = response ?? '';
 
-        this.cartId.setValue(this.activeCartId);
+        this.cartId.setValue(
```

Comment:
maybe in here always show cartID from URL.

Reply:
fixed